plugin.go

```diff
@@ -78,12 +79,36 @@ func (p *Plugin) Serve() chan error {
 
 			for i := 0; i < len(procs); i++ {
 				cmdStr := procs[i].service.Command
-				err := procs[i].start()
-				if err != nil {
-					errCh <- err
-					return false
+
+				if procs[i].service.StartDelaySec > 0 {
+					delayDuration := time.Duration(procs[i].service.StartDelaySec) * time.Second
+					p.logger.Info("service will start with delay",
+						zap.String("name", key.(string)),
+						zap.String("command", cmdStr),
+						zap.Uint64("delay_seconds", procs[i].service.StartDelaySec))
+
+					go func(proc *Process, name string, cmd string) {
+						time.Sleep(delayDuration)
```

Comment:
This is not a good way to implement delays:
1. How to stop the service if I hit ctrl-c faster than delay? E.g.: delay was set to 10 seconds, but I'll hit ctrl-c after 5 seconds?
2. Service should be implemented in a way to stop all delayed goroutines.
3. Delay can be implemented in PHP script, why it should be in services plugin?